src/components/ManagerPage/AppsList.js

```diff
@@ -145,12 +141,12 @@ class AppsList extends PureComponent<Props, State> {
   _unmounted = false
 
   async fetchAppList() {
-    const { deviceInfo, isDevMode } = this.props
+    const { deviceInfo } = this.props
 
     try {
       const filteredAppVersionsList = await manager.getAppsList(
         deviceInfo,
-        isDevMode,
+        getEnv('MANAGER_DEV_MODE'),
```

Comment:
quick thought in the future: we might introduce a `useEnv("MANAGER_DEV_MODE")` to use with react hooks. without react hooks, it would be a `withEnvs` hoc i guess. It would trigger a re-render when env changes (with the observable we have from live-common).

here we really have no problem because the manager screens can't appear at same time as the settings. However on mobile we need it since all screens are always mounted.

Reply:
hooks <3